src/utils/ddb/tests/ddb_parse_tests.c

```diff
@@ -40,7 +40,6 @@ assert_parsed_fail(const char *str)
 	int			rc;
 
 	rc = ddb_str2argv_create(str, &parse_args);
-	ddb_str2argv_free(&parse_args);
```

Comment:
Seems unnecessary since you have changed ddb_str2argv_free() with checking "if (parse_args->ap_ctx != NULL) {" firstly.

Reply:
True, but I just want to make sure I resolve the Coverity issues. :-)